community/samples/README.md

```diff
@@ -0,0 +1,3 @@
+# Knative Community Samples
```

Comment:
Can we add links to the community samples from here, similar to the /serving/samples/README.md?

Reply:
done.

Reply:
Thanks!